web/react/stores/browser_store.jsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+var UserStore = require('../stores/user_store.jsx');
+var BROWSER_STORE_VERSION = '.1';
```

Comment:
I wonder if this should be in-sync with the etag root version on the backend?  Maybe add a comment to update in both places.